### DISCUSSION TITLE
Using MAST for ATAC data

### DISCUSSION THREAD

**Discussion by AmelZulji** [stuart-lab/signac#1338] [Q&A]
Hello Signac developers and community, 

I was wondering whether it is good idea to use MAST (not as part of FindMarkers(... test.use = "MAST"), but standalone package) for calculating differentially accessible peaks. I am having 10x multiome data and for gene expression readout I am very happy with the following model: 
`zlmCond <- zlm(~sex + cond + (1| library_id), sca =  sca, method = 'glmer', ebayes = F, parallel = T)` 
where `library_id` is sample and `cond` is condition.

Would something similar work and produce reliable results for ATAC data? If not, are there any tools which can enable a bit more complex designs for ATAC data? 

I am grateful for your help! 

Amel

**Comment by timoast** [comment]
Hi, I'd suggest asking the MAST developers: https://github.com/RGLab/MAST
